service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/DiaryPlanStatementMapperTest.java

```diff
@@ -193,6 +193,95 @@ public void When_MappingWithOccurrenceWithOnlyStart_Expect_TextDoesNotContainEar
         assertThat(actualXml).isEqualTo(expectedXml);
     }
 
+    @Test
+    public void When_MappingWithDeviceReferenceWhereDeviceHasManufacturer_Expect_TextContainsDeviceTypeTextAndManufacturer() {
+
+        var inputJson = """
+            {
+                "resourceType": "ProcedureRequest",
+                "id": "1B1C6F50-D8BE-4480-83D6-EF25AC5F1836",
+                "status": "active",
+                "intent": "plan",
+                "authoredOn": "2010-01-13T15:29:50.1+00:00",
+                "requester": {
+                    "agent": {
+                        "reference": "Device/6D340A1B-BC15-4D4E-93CF-BBCB5B74DF73"
+                    }
+                },
+                "code": {
+                    "text": "test"
+                }
+            }""";
+
+        final var expectedXml = """
+            <component typeCode="COMP" >
+                <PlanStatement classCode="OBS" moodCode="INT">
+                    <id root="394559384658936" />
+                    <code nullFlavor="UNK"><originalText>Mocked code</originalText></code>
+                    <text>
+                        Recall Device: Device type text Device name
+                    </text>
+                    <statusCode code="COMPLETE" />
+                    <effectiveTime>
+                        <center nullFlavor="UNK"/>
+                    </effectiveTime>
+                    <availabilityTime value="20100113152950"/>
+                </PlanStatement>
+            </component>""";
+
+        final var procedureRequest = new FhirParseService().parseResource(inputJson, ProcedureRequest.class);
+        var actualXml = diaryPlanStatementMapper.mapDiaryProcedureRequestToPlanStatement(procedureRequest, false);
+
+        assertThat(actualXml).isEqualTo(expectedXml);
+    }
+
+    @Test
+    public void When_MappingWithDeviceReferenceWhereDeviceHasNoManufacturer_Expect_TextOnlyContainsDeviceTypeText() {
+        String inputBundle = ResourceTestFileUtils.getFileContent(
+            TEST_DIRECTORY + "input-bundle-no-manufacturer.json"
+        );
+        Bundle bundle = new FhirParseService().parseResource(inputBundle, Bundle.class);
+        messageContext.initialize(bundle);
+
+        var inputJson = """
+            {
+                "resourceType": "ProcedureRequest",
+                "id": "1B1C6F50-D8BE-4480-83D6-EF25AC5F1836",
+                "status": "active",
+                "intent": "plan",
+                "authoredOn": "2010-01-13T15:29:50.1+00:00",
+                "requester": {
+                    "agent": {
+                        "reference": "Device/6D340A1B-BC15-4D4E-93CF-BBCB5B74DF73"
```

Comment:
Not necessarilly a blocker, but if it were me I'd have added to the existing `input-bundle.json` a new Device, with an `id` of "With-Null-Manufacturer" or similar, and then referenced from here.

Means don't have to maintain a separate `input-bundle-no-manufacturer.json`, and also helps improve the self-documentation-ness of the `ProcedureRequest test. "Device/With-Null-Manufacturer" helps guide the person who's read the test name to follow what's going on.